modules/serverless-https-redirect-global.adoc

```diff
@@ -0,0 +1,35 @@
+// Module is included in the following assemblies:
+//
+// * serverless/admin_guide/knative-serving-CR-config.adoc
+
+[id="serverless-https-redirect-global_{context}"]
+= HTTPS redirection global settings
+
+You can enable HTTPS redirection for all services on the cluster, by configuring the `httpProtocol` spec for the `KnativeServing` custom resource, as shown in the following example:
+
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+spec:
+  config:
+    network:
+      httpProtocol: "enabled"
```

Comment:
Thank you and nice catch. Yes, we should use `redirected` instead of `enabled` in this example as we are talking about how to enable HTTPS redirection.

Reply:
@nak3 can you please also open an issue or PR to fix this in the upstream docs

Reply:
Sure, opened https://github.com/knative/docs/pull/4510